vhs-frontend/src/pages/Explore.tsx

```diff
@@ -1,68 +1,49 @@
-import { useEffect, useState } from "react";
-import axios from "axios";
-import { VHS } from "../types";
-import { VhsThumbnail } from "../components/VHSThumbnail";
-import { SearchBar } from "../components/SearchBar";
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { VHS } from '../types';
+import { VhsThumbnail } from '../components/VHSThumbnail';
+import { SearchBar } from '../components/SearchBar';
 // @ts-ignore
-import placeholder from "../assets/placeholder.jpg";
+import placeholder from '../assets/placeholder.jpg';
+import { SearchFormInput } from '../components/searchBarConfig';
 
 export const Explore = () => {
   const [vhsList, setVhsList] = useState<VHS[]>([]);
-  const [filteredList, setFilteredList] = useState<VHS[]>([]);
-  const [searching, setSearching] = useState(false);
+  const [query, setQuery] = useState('');
 
   useEffect(() => {
-    getVhsList();
-  }, []);
+    getVhsList(query);
+  }, [query]);
 
-  const getVhsList = async () => {
+  const getVhsList = async (query?: string) => {
     try {
-      const response = await axios.get("/api/vhs");
+      const response = await axios.get(`/api/vhs${query}`);
       setVhsList(response.data);
     } catch (error) {
       console.error(error);
     }
   };
+
+  const onSubmit: (data: SearchFormInput) => void = (data) => {
+    const query = `?${data.queryType}=${data.queryText}`;
```

Comment:
If you change how you use `axios.get` with query params, you will need to change this as well. The query you save in the state will be an object with queryType and queryValue properties, not an interpolated string (axios will do that itself).